### PR TITLE
OMX Mapping + Network validation scripts + cosmetic changes

### PR DESCRIPTION
**Pull request by haitamlaarabi**
Original PR: #3855

This change is [Reviewable](https://reviewable.io/reviews/LBNL-UCB-STI/beam/3856)

### CODE REVIEW

**Comment by nikolayilyin** [comment]
test!

File: src/main/scala/beam/router/skim/ActivitySimSkimmer.scala [hunk 1]
```diff
@@ -240,8 +240,8 @@ class ActivitySimSkimmer @Inject() (matsimServices: MatsimServices, beamScenario
     )
 
     val mappedSkimFilePath = filePath.lastIndexOf(".") match {
-      case -1    => filePath + "_mapped" // No extension found, append "_mapped" at the end
-      case index => filePath.substring(0, index) + "_mapped" + filePath.substring(index)
+      case -1    => filePath + "_geo_exchange" // No extension found, append "_mapped" at the end
+      case index => filePath.substring(0, index) + "_geo_exchange" + filePath.substring(index)
```
Comment (nikolayilyin) [inline src/main/scala/beam/router/skim/ActivitySimSkimmer.scala]: redundant comment - let's remove it

**Review by nikolayilyin — APPROVED** [review]

**Review by nikolayilyin — APPROVED** [review]

**Comment by nikolayilyin** [comment]
test now!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test now!

**Comment by nikolayilyin** [comment]
test now!

**Comment by haitamlaarabi** [comment]
test!

**Review by nikolayilyin — APPROVED** [review]
lgtm

**Comment by nikolayilyin** [comment]
test now!

**Comment by nikolayilyin** [comment]
test!

**Comment by haitamlaarabi** [comment]
test now!

**Comment by haitamlaarabi** [comment]
test now!

**Comment by haitamlaarabi** [comment]
test!

**Comment by haitamlaarabi** [comment]
test now!

**Review by nikolayilyin — APPROVED** [review]
lgtm

**Review by nikolayilyin — APPROVED** [review]
> Reviewed 13 of 31 files at r1, 5 of 18 files at r5, 4 of 9 files at r6, 5 of 8 files at r7, 8 of 8 files at r8, 2 of 2 files at r9, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/LBNL-UCB-STI/beam/3856)* status: all files reviewed, 1 unresolved discussion (waiting on @dimaopen, @haitamlaarabi, and @zneedell)

**Review by nikolayilyin — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/LBNL-UCB-STI/beam/3856)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @dimaopen and @zneedell)

**Pull request merged**